google/cloud/storage/internal/credential_constants.h

```diff
@@ -36,9 +36,9 @@ enum class JwtSigningAlgorithms { RS256 };
 inline char const* GoogleOAuthRefreshEndpoint() {
   // TODO(#769): Transition to using the new audience endpoint:
   // https://oauth2.googleapis.com/token
-  static constexpr char endpoint[] =
+  static constexpr char kEndpoint[] =
```

Comment:
Can you create a GitHub issue to enforce this via clang-tidy?

Reply:
Filed #894.